src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/DynamicButtonsSimpleTest.java

```diff
@@ -0,0 +1,23 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+import catalinazoldi.tests.Tema8.pageObjects.DynamicButtonsSimplePage;
```

Comment:
Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.DynamicButtonsSimplePage;`